Project-3/ArunBhagat204/README.md

```diff
@@ -0,0 +1,51 @@
+# README
```

Comment:
Please add a link to your website and optionally screenshots of the functionality here itself.

Thanks!

Reply:
Bhaiya, I created added the screenshots in a separate folder but how do I provide the link to the website, I have not hosted it anywhere, you can access it using the 'index.html' file.

Reply:
I added the link to the README file and the screenshots are now available in the 'Screenshots' folder.